internal/connect/client.go

```diff
@@ -346,6 +350,13 @@ func DeactivateProduct(product Product) (Service, error) {
 
 // DeregisterSystem deletes current system in SMT/SCC
 func DeregisterSystem() error {
```

Comment:
Preliminary test looks good. Testing it on `registry.suse.com/suse/sle15:15.4` and found the credentials written in `/root/.docker/config.json`. Deregistering doesn't remove the credentials, see below comment.

Reply:
Jep, this and the ownership of the files is wrong. While `docker` config is fine, `podman` config should follow the ownership of `%XDG_RUNTIME_DIR` and not the calling user (aka. root). Will fix!

Reply:
moved!